installing/installing-fips.adoc

```diff
@@ -71,6 +71,7 @@ To ensure that containers know that they are running on a host that is using FIP
 To install a cluster in FIPS mode, follow the instructions to install a customized cluster on your preferred infrastructure. Ensure that you set `fips: true` in the `install-config.yaml` file before you deploy your cluster.
 
 * xref:../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[Amazon Web Services]
+* xref:../installing/installing_alibaba/preparing-to-install-on-alibaba.adoc#preparing-to-install-on-alibaba[Alibaba Cloud]
```

Comment:
Suggest to be consistent with other providers' links, i.e. use `installing-alibaba-customizations.html` instead. Thanks!

Reply:
Thanks, @jianli-wei , for the feedback. I've updated the link as requested.